crates/downloader/src/config.rs

```diff
@@ -0,0 +1,33 @@
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct Config {
+    /// URL to the Ethereum RPC endpoint.
+    /// For example `https://sepolia.infura.io/v3/API_KEY_HERE`.
+    pub ethereum_rpc_url: String,
+    /// URL to the Ethereum consensus/beacon layer RPC endpoint.
+    /// For example `https://ethereum-sepolia.core.chainstack.com/beacon/API_KEY_HERE`.
+    pub beacon_rpc_url: String,
+    /// Contract to monitor for new blobs.
+    /// For example `0xB0B3682211533cB7C1a3Bcb0e0Dd4349fF000d75`.
+    pub blob_contract: String,
```

Comment:
Hmm, I though that we need to monitor the address that posts blobs, not the contract

Reply:
I don't understand the disctinction here. Maybe @segfault-magnet knows?